src/common/actions/snap-builds.js

```diff
@@ -59,7 +60,13 @@ export function fetchSnap(repository) {
       return fetch(url)
         .then(checkStatus)
         .then(response => response.json())
-        .then((json) => dispatch(fetchBuilds(json.payload.message)))
```

Comment:
`fetchSnap` is not starting fetching builds anymore, just triggers an action to   save snap link in the store.

`fetchBuilds` is called directly from builds container once snap link is in the store